tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -210,9 +222,21 @@ namespace.ceph:
           enabled: true
           inputs:
             mandatory:
-              - Rbd.pool_id
               - Rbd.name
               - Rbd.size
+            optional:
```

Comment:
@r0h4n @anmolsachan For now, the UI is passing Rbd.pool_id, Rbd.name and Rbd.size which are the mandatory fields for create RBD flow.

Create Rbd flow using new pool as the backing pool is not yet supported in the UI(Issue - https://github.com/Tendrl/api/issues/153). I think these optional fields will come into picture when UI is able to create rbd using new pool.

But as per the pool creation design and the current implementation, UI can send the following parameters  to API - 
1) pool_type (optional)
2) pool_size (mandatory)
3) pool_pg_num (mandatory)
4) pool_poolname (mandatory)
5) pool_min_size (mandatory)
6) pool_quota_enabled (optional)
7) pool_quota_max_objects (optional).